src/components/UI/Display/MAPI/AccessControl/types.ts

```diff
@@ -44,21 +44,27 @@ export interface IAccessControlRoleItem {
 }
 
 export interface IAccessControlSubjectPermissions {
-  canAdd: boolean;
+  canCreate: boolean;
+  canBind: boolean;
   canDelete: boolean;
   canList: boolean;
 }
 
+export interface IAccessControlRolePermissions {
+  canList: boolean;
+}
+
 export interface IAccessControlPermissions {
```

Comment:
This wouldn't be a good example for handling permissions. But since the access control part of the UI is built differently compared to other parts of the app, we need to get dirty with it